CLAUDE.md

```
@@ -24,25 +24,55 @@ Hosted on Cloudflare Pages.
-Blog posts support versioning for edit history tracking.
+Blog posts and projects support versioning for edit history tracking.
+
+### Core Concept
+
+- `live.mdx` is the **single source of truth** — it has the full git history for the post
+- Snapshots are created at significant milestones (not every edit)
+- Minor edits (typos, formatting) are committed directly to `live.mdx` without a snapshot
+- You decide on a case-by-case basis when a snapshot is warranted
 
 ### File Structure
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)